src/Umbraco.Forms.Integrations.Commerce.EMerchantPay/Models/Dtos/NotificationDto.cs

```diff
@@ -1,15 +1,27 @@
-﻿using System.ComponentModel;
-using System.Runtime.Serialization;
-using System.Xml.Serialization;
-
+﻿
 namespace Umbraco.Forms.Integrations.Commerce.EMerchantPay.Models.Dtos
 {
     public class NotificationDto
     {
-        public string TransactionId { get; set; }
+        /// <summary>
```

Comment:
I wonder here if you could do this using an attribute?  I.e. just have the properties `TransactionId` and `UniqueId`, and use an attribute to map them when serializing from the field names in the posted data (i.e. `wpf_transaction_id` and `wpf_unique_id`).  

It would be this I believe if expecting XML data:

```
[XmlElement(ElementName = "wpf_transaction_id")]
public string TransactionId { get; set; }
```